docs/contributor/benchmarks/README.md

```diff
@@ -118,9 +118,12 @@ A typical test result output looks like the following example:
 |            0.107.0 |            19876            |            19719            |          0          |       184, 96        |     1, 1      |            11993             |            35504            |          0          |       133, 138       |   1.4, 1.4    |                   10081                    |             360             |         512         |      1734, 1722      |   0.5, 0.5    |                   18184                    |             922             |         510         |      1754, 1698      |    0.9, 1     |
 |            0.108.1 |            20175            |            20014            |          0          |       100, 93        |     1, 1      |            12365             |            36603            |          0          |       135, 148       |   1.4, 1.4    |                   10758                    |             332             |         512         |      1722, 1681      |   0.5, 0.5    |                   19163                    |             891             |         510         |      1800, 1757      |     1, 1      |
 
-
 </div>
 
+> **NOTE:** The expected throughput for the TracePipeline receiver on the receiver side is ~20K spans/sec for the single-pipeline scenario and ~13K spans/sec for the multi pipeline scenario. Exported data should be 3 times the received spans. The backpressure scenarios should return the exporter queue size as ~512, for both single- and multi-pipeline scenarios; and the TracePipeline instances should not crash.
```

Comment:
I think an info like following is acceptable range would make this point more clear